cli/deployments/quarks.go

```diff
@@ -45,8 +47,32 @@ func (k Quarks) Describe() string {
 	return emoji.Sprintf(":cloud:Quarks version: %s\n:clipboard:Quarks chart: %s", quarksVersion, quarksChartURL)
 }
 
+// Delete removes Quarks from kubernetes cluster
 func (k Quarks) Delete(c kubernetes.Cluster) error {
-	return c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), quarksDeploymentID, metav1.DeleteOptions{})
+	message := "Deleting Quarks"
+	s := spinner.New(spinner.CharSets[11], 100*time.Millisecond)
+	s.Suffix = emoji.Sprintf(" %s :zzz:", message)
+	s.Start()
+
+	currentdir, err := os.Getwd()
+	if err != nil {
+		return errors.New("Failed uninstalling Quarks: " + err.Error())
+	}
+
+	helmCmd := fmt.Sprintf("helm uninstall quarks --namespace %s", quarksDeploymentID)
+	if out, err := helpers.RunProc(helmCmd, currentdir, k.Debug); err != nil {
+		return errors.New("Failed uninstalling Quarks: " + out)
+	}
+
+	err = c.Kubectl.CoreV1().Namespaces().Delete(context.Background(), quarksDeploymentID, metav1.DeleteOptions{})
+	if err != nil {
+		return errors.New("Failed uninstalling Quarks: " + err.Error())
+	}
+	s.Stop()
```

Comment:
Move up to `s.Start()` and use `defer` to ensure execution regardless of code path.

Reply:
Done with commit 84de7f0c9c6094ee525ad5b8033ae42f11225d03.